diagnostics/app/model/diagnostics/DynamoDbReport.scala

```diff
@@ -72,7 +72,7 @@ object CSPDynamoDbReport extends DynamoDbReport[CSPReport] {
       .withAttributeUpdates(count.asJava)
 
     client.updateItemFuture(update) onFailure {
-      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB", error)
+      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB: $error", error)
```

Comment:
Does that work? $error isnt a string ... ${error.getMessage] mebbe. You don't need another arg for string interpolation
Or am I being **really** stupid?